opentelemetry-sdk/src/metrics/test.md

```diff
@@ -0,0 +1,24 @@
+# Test Coverage for Metric Instruments
+
+Tests are located in [mod.rs](mod.rs)
+
+// TODO: Fill this correctly.
+
+## Sync Instruments
+
+| Test Type                  | Counter (Delta) | Counter (Cumulative) | UpDownCounter (Delta) | UpDownCounter (Cumulative) | Gauge (Delta) | Gauge (Cumulative) | Histogram (Delta) | Histogram (Cumulative) |
+|----------------------------|-----------------|----------------------|----------------------|----------------------------|---------------|--------------------|-------------------|------------------------|
+| Regular aggregation test   | [Yes]         | [Yes]              | [No]              | [No]                    | [No]      | [No]            | [No]           | [No]                |
+| No-attribute test          | [Yes]         | [Yes]              | [No]              | [No]                    | [No]      | [No]            | [No]           | [No]                |
+| Overflow test              | [Yes]         | [No]               | [No]              | [No]                    | [No]      | [No]            | [No]           | [No]                |
+| Attr Order Sorted First    | [Yes]         | [Yes]              | [No]              | [No]                    | [No]      | [No]            | [No]           | [No]                |
+| Attr Order Unsorted First  | [Yes]         | [Yes]              | [No]              | [No]                    | [No]      | [No]            | [No]           | [No]                |
```

Comment:
Need to add entries for multi-threaded test and other data types such as `f64` and `i64`.

Reply:
Also, need to add entries for special scenarios such as testing the export results when no new measurement has been emitted since the last collect.